packit/local_project.py

```diff
@@ -12,6 +12,25 @@
 
 
 class LocalProject:
+    """
+    Class representing the cloned repository and the remote API
```

Comment:
```suggestion
    Class representing a cloned repository and the corresponding remote API bound together
```

Reply:
I've changed it a bit and mention `Github/Gitlab/Pagure` to be really explicit....